apps/snitch_core/lib/core/tools/helpers/date_formatter.ex

```diff
@@ -0,0 +1,10 @@
+defmodule Snitch.Tools.Helper.DateFormatter do
+  # defmacro __using__(_) do
```

Comment:
This module is supposed to contain all the helpers related to fragments I think one of which is `to_char` this should not be name DateFormatter, also you haven't used `__using__(_)` why leave commented code here. But, why are you not using `__using__(_)`